README.md

```diff
@@ -210,13 +210,64 @@ Key functions available (see JSDoc comments in source for detailed documentation
 
+
+## Theming
+
+Customizing the theme in this project is straightforward and leverages Bulma's powerful theming capabilities. You can override Bulma's default variables to match your brand or design requirements by editing the `frontend/src/styles/main.scss` file. This is done **before importing Bulma** so that your customizations are applied throughout the project.
+
+### Steps to Customize
+
+1. Navigate to the `frontend/src/styles/main.scss` file.
+2. Locate the section for overriding Bulma variables.
+3. Uncomment and modify the variables you wish to customize. You can define colors, typography, spacing, and more.
+4. Save your changes, and the updated theme will automatically apply when you rebuild the project.
+
+### Example: Overriding Common Variables
+
+Here’s an example of how to customize some of Bulma's common variables. These variables are commented out by default. Uncomment and modify them as needed:
+
+```scss
+@use 'bulma/sass/utilities/initial-variables' as * with (
+  // Colors
+  $turquoise: hsl(171, 100%, 41%),   // Primary color
+  $cyan: hsl(204, 86%, 53%),         // Info color
+  $green: hsl(141, 71%, 48%),        // Success color
+  $yellow: hsl(48, 100%, 67%),       // Warning color
+  $red: hsl(348, 100%, 61%),         // Danger color
+
+  // Typography
+  $family-sans-serif: BlinkMacSystemFont, -apple-system, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", "Helvetica", "Arial", sans-serif,
+  $family-monospace: monospace,
+  $size-1: 3rem,
+  $size-2: 2.5rem,
+  $size-3: 2rem,
+  $size-4: 1.5rem,
+  $size-5: 1.25rem,
+  $size-6: 1rem
+);
+```
+
+### Full List of Variables
+For a comprehensive list of all customizable variables, refer to the [Bulma documentation](https://bulma.io/documentation/customize/list-of-sass-variables/) on variables. This resource provides details on all available options for customization, including advanced options for responsive breakpoints, spacing, and more.
+
+### Notes
+* **Order matters**: Ensure your variable overrides are declared before importing Bulma to avoid conflicts.
+* **SASS compatibility**: This setup uses the modern SASS syntax with @use and @forward. If you are unfamiliar with these concepts, refer to the SASS documentation for more information.
+* **Theme consistency**: To maintain a cohesive design, consider defining your core color palette and typography styles at the beginning of your project.
+
+### Troubleshooting
+If your changes are not reflected:
+
+* Ensure your variables are correctly uncommented and modified.
+* Check for any caching issues by clearing your browser cache or restarting the build process.
+
 # Deploying to production
 
 ## Using Apostrophe hosting
 
 Apostrophe can provide easy hosting for any ApostropheCMS-Astro monorepo with little or no extra configuration. This can be setup for deployment from Github or other code repository.
```

Comment:
setup -> set up

So we're holding this back until we release workspaces hosting?

Reply:
Fixed and hosting situation updated